sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientBase.java

```diff
@@ -129,7 +129,7 @@ public abstract class IdentityClientBase {
     private static final ClientOptions DEFAULT_CLIENT_OPTIONS = new ClientOptions();
 
 
-    private static final Map<String, String> PROPERTIES = CoreUtils.getProperties(AZURE_IDENTITY_PROPERTIES);
+    private final Map<String, String> properties = CoreUtils.getProperties(AZURE_IDENTITY_PROPERTIES);
```

Comment:
why make this non static ? 
doesn't need to be instantiated per instance.

Reply:
It previously was non-static before I made the change that exposed `setupPipeline`. See [here](https://github.com/Azure/azure-sdk-for-java/pull/40246/files#diff-b5937fc0e636a1c6954f1b77ed7149066d1f7048dd85d8db025609d2615c514cR130). I just rolled back all the things involved in making `setupPipeline` a static usable outside the class.